### DISCUSSION TITLE
Como registar as faturas de importação no SAF-T das compras?

### DISCUSSION THREAD

**Discussion by cryptolopes** [assoft-portugal/SAF-T-AO#220] [Q&A] [answered]
Qual é a forma adequada de registar as faturas de importação do SAF-T (AO) das compras?
De notar que estas faturas não têm IVA.
O que considerar, a fatura do fornecedor ou o documento aduaneiro?

**Comment by Schobert** [comment, marked as answer]
Como combinado na nossa reunião, aqui vai o meu resumo:
De acordo com o esclarecimento que tivemos com a AGT há 3 anos atrás:
Na importação temos 2 documentos, a fatura da importação e o o documento de liquidação da alfândega (normalmente emitido pelo despachante, eventualmente junto com o serviço prestado de transporte).

Valor da importação 1000,00; IVA reverse charge 14% (pago na alfândega); 100% dedutível
**Fatura com IVA autoliquidado;** O IVA reportado é a parte dedutível: 

InvoiceNo>1234567</InvoiceNo>
.....
DocumentTotals>
TaxPayable>140.00</TaxPayable>
NetTotal>1000.00</NetTotal>
GrossTotal>1140.00</GrossTotal>
Deductible>
TaxBase>1000.00</TaxBase>
DeductibleAmount>140.00</DeductibleAmount>
DeductiblePercentage>100.00</DeductiblePercentage>
DeductibleTaxType>IMPT</DeductibleTaxType>

**Se houver Documento de liquidação**

InvoiceNo>203</InvoiceNo>
....
DocumentTotals>
TaxPayable>140.00</TaxPayable>
NetTotal>0.00</NetTotal>
GrossTotal>140.00</GrossTotal>
Deductible>
TaxBase>1000.00</TaxBase>
DeductibleAmount>140.00</DeductibleAmount>
DeductiblePercentage>100.00</DeductiblePercentage>
